src/components/Education.js

```diff
@@ -1,34 +1,131 @@
+              className="school"
+              id="school"
+              value={schoolData.school}
+              onChange={(e) => {
+                onChange('education', {
+                  ...schoolData,
+                  school: e.target.value,
+                });
+              }}
+            />
+          </label>
+          <label htmlFor="title">
+            Title of studies
+            <input
+              className="title"
+              id="title"
+              value={schoolData.title}
+              onChange={(e) => {
+                onChange('education', {
+                  ...schoolData,
+                  title: e.target.value,
+                });
+              }}
+            />
+          </label>
+          <label htmlFor="date">
+            Date
+            <label htmlFor="date">
+              From
+              <input
+                type="month"
+                className="date"
+                id="date"
+                value={schoolData.date.from}
+                onChange={(e) => {
+                  onChange('education', {
+                    ...schoolData,
+                    date: {
+                      ...schoolData.date,
+                      from: e.target.value,
+                    },
+                  });
+                }}
+              />
+            </label>
+            <label htmlFor="date">
+              To
+              <input
+                type="month"
+                className="date"
+                id="date"
+                value={schoolData.date.to}
+                onChange={(e) => {
+                  onChange('education', {
+                    ...schoolData,
+                    date: {
+                      ...schoolData.date,
+                      to: e.target.value,
+                    },
+                  });
+                }}
+              />
+            </label>
+          </label>
+          <hr />
+        </div>
+      );
+    }
+
+    return content;
+  }
 
-export default function Education() {
   return (
     <div className={styles.container}>
       <div className={styles.header_container}>
         <h1>Education</h1>
-        <div className="editButton" onClick={handleClick}>
-          Edit
+        <div className={styles.editButton} onClick={handleClick}>
+          {isEditing ? 'Close editing' : 'Edit'}
         </div>
       </div>
-      <label htmlFor="school">
-        School
-        <div className="school" id="school">
-          School
-        </div>
-      </label>
-      <label htmlFor="title">
-        Title of studies
-        <div className="title" id="title">
-          Errr
-        </div>
-      </label>
-      <label htmlFor="date">
-        Date
-        <div className="date" id="date">
-          2000
-        </div>
-      </label>
+      {data.map((job, index) => (
+        <Fragment key={index}> {generateContent(job)}</Fragment>
+      ))}
+      <div className={styles.addInfo} onClick={onClick}>
```

Comment:
Try to stick to using forms when doing input changes and submit. Then you can use onSubmit on <form> rather than custom onClick. With <form> you can manage validation easier also.